lib/pwa/offline_urls.g.dart

```diff
@@ -29,4 +29,8 @@ final List<String> offlineUrls = [
   './packages/test/src/runner/browser/static/',
   './packages/test/src/runner/browser/static/host.css',
   './packages/test/src/runner/browser/static/host.dart.js',
+  './pwa.dart.info.json',
```

Comment:
Right. On top of that, the `./packages/test/src/runner/browser/static/host.css` and its `host.dart.js`, and the `./game_web.dart.info.json` should be also removed from this list. I'm a bit undecided whether these should be in the default list of `pwa`, or manually filtered with command line flags (+ documenting it). Probably it should be part of the `pwa`, let me know if you think otherwise.